ui/components-react/src/components-react/properties/renderers/PropertyView.tsx

```diff
@@ -113,6 +116,7 @@ export class PropertyView extends React.Component<
         {needElementSeparator ? (
           // eslint-disable-next-line deprecation/deprecation
           <ElementSeparator
+            style={{ margin: 0 }}
```

Comment:
Looks good. This is needed so it doesn't apply the negative margins? This is just what I gathered from storybook. Ignore if it doesn't make sense

Reply:
Yes, I was hesitant to change it at element separator level due to unknown consequences.